core/lib/protobuf_config/src/secrets.rs

```diff
@@ -103,14 +103,37 @@ impl ProtoRepr for proto::DataAvailabilitySecrets {
         let secrets = required(&self.da_secrets).context("config")?;
 
         let client = match secrets {
-            DaSecrets::Avail(avail_secret) => DataAvailabilitySecrets::Avail(AvailSecrets {
-                seed_phrase: Some(
-                    SeedPhrase::from_str(
-                        required(&avail_secret.seed_phrase).context("seed_phrase")?,
+            DaSecrets::Avail(avail_secret) => {
+                let seed_phrase = if avail_secret.seed_phrase.is_some() {
+                    Some(
+                        SeedPhrase::from_str(
+                            required(&avail_secret.seed_phrase).context("seed_phrase")?,
+                        )
```

Comment:
it looks weird to check if it's some and then use `required`, please refactor. Same for `gas_relay_api_key` below

Reply:
Fixed, closing.